django_prbac/arbitrary.py

```diff
@@ -2,8 +2,8 @@
 from __future__ import unicode_literals, absolute_import, print_function
 
 # Standard Library Imports
-from random import choice
 import uuid
+from random import choice
```

Comment:
Is it normal to have `uuid` before `random` since it's import vs from? I've always just done them alphabetically, not sure what's right

Reply:
I believe the convention is to do `import ...` before `from ... import ...`. PEP8 is not very explicit about that though.